public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/__snapshots__/SimplifiedRuleEditor.test.tsx.snap

```diff
@@ -79,7 +79,9 @@ exports[`Can create a new grafana managed alert using simplified routing can cre
                         "type": "and",
                       },
                       "query": {
-                        "params": [],
+                        "params": [
```

Comment:
@L2D2Grafana looks like an unintentional change from the rebase? I think this is what's breaking the unit tests too